packages/visualizations/src/Legend.tsx

```diff
@@ -49,7 +57,7 @@ export const Legend = (props: Props<string>) => {
   return <div style={{...props.style, ...legendStyle}}>
     {props.title && <div style={titleStyle}>{props.title}</div>}
     {props.data.groupBy(props.cursors).map((grouped, i) =>
-      <div style={itemStyle} key={i}>
+      <div style={itemStyle(props.itemWidth)} key={i}>
```

Comment:
```suggestion
      <div style={itemStyle(props.itemWidth)} key={i} title={joinArrayAsString(uniqueValues[i])}>
```

(optional) this will show small tooltip if you hover mouse, in case item is cut by ellipsis

Reply:
Ah, great idea!